cli/azd/pkg/exec/cmdtree_windows.go

```diff
@@ -79,3 +125,35 @@ func (o *CmdTree) Kill() {
 		log.Printf("failed to terminate job object %d: %s\n", o.jobObject, err)
 	}
 }
+
+// buildCmdCmdLine builds the command line for`cmd.exe` to have it run command or list of commands. When cmd is non empty
+// it is treated as the command to run and args is the array of arguments for cmd. Otherwise, each arg is treated as an
+// individual command, and joined with &&.  Each individual command is quoted.
+func buildCmdCmdLine(cmd string, args []string) string {
```

Comment:
In general -- separate from this change, not a huge fan of the shell abstraction, and wonder if we're better off just providing a utility to format shell arguments and build the cmd command for a given shell.

Invoking a shell to launch a program has very different rules than launching a program directly due to shell evaluation rules on the parameters. Which means that the consequences of doing one versus another is huge enough that I don't quite like the uniformity in the API. Escaping rules are also shell-specific which further breaks encapsulation.

Just bringing this up in case you noticed any further need here to reevaluate the design. Happy to land this to solve the user-reported issue.